tests/MongoDB.Driver.Legacy.Tests/MongoCollectionTests.cs

```diff
@@ -423,6 +427,26 @@ public void TestCountZero()
             Assert.Equal(0, count);
         }
 
+        [SkippableFact]
+        public void TestCountUsesImplicitSession()
```

Comment:
We could add new tests for *every* operation but they all funnel through the same UsingImplicitSession method so I'm hoping this one test will suffice.